README.md

```diff
@@ -1,42 +1,29 @@
-
-## Documentation
-Full documentation is available at [GitHub Pages](https://curricularanalytics.github.io/CurricularAnalytics.jl/latest/).
-Documentation for functions in this toolbox is also available via the Julia REPL help system.
-Additional tutorials can be found at the [Curricular Analytics Notebooks](https://github.com/CurricularAnalytics/CA-Notebooks) site.
-
-# Installation
-
-Installation is straightforward.  First, install the Julia programming language on your computer.  To do this, download Julia here: https://julialang.org, and follow the instructions for your operating system.
-
-Next, open the Julia application that you just installed. It should look similar to the image below. This interface is referred to as the `Julia REPL`.
-
-![Julia termain](https://s3.amazonaws.com/curricularanalytics.jl/julia-command-line.png)
-
-Next, enter Pkg mode from within Julia by hitting the `]` key, and then type:
-```julia-repl
-  pkg> add CurricularAnalytics
-```
-This will install the toolbox, along with the other Julia packages needed to run it. To load and use the toolbox, hit the `backspace` key to return to the Julia REPL. Now type:
-```julia-repl
-  julia> using CurricularAnalytics
-```
-The toolbox must be loaded again via `using CurricularAnalytics` every time you restart Julia.
-
-For more information about installing the toolbox, including the steps neccessary to utilize degree plan optimization, please see the [INSTALLING.md](https://curricularanalytics.github.io/CurricularAnalytics.jl/latest/install.html)
-
-## Supported Versions
-* CurricularAnalytics master will be maintained/enhanced to work with the latest stable version of Julia.
-* Julia 1.0.5, 1.2, 1.3, 1.4: CurricularAnalytics v1.0.0 is the latest version guaranteed to work with Julia 1.0.5 to version 1.4.
-* Later versions: Some functionality might not work with prerelease / unstable / nightly versions of Julia. If you run into a problem, please file an issue.
-
-# Contributing and Reporting Bugs
-We welcome contributions and bug reports! Please see [CONTRIBUTING.md](https://github.com/CurricularAnalytics/CurricularAnalytics.jl/blob/master/CONTRIBUTING.md)
-for guidance on development and bug reporting.
+# CurricularAnalytics.jl (Simulation)
+
+## Installation
+- Clone the repo to the local machine
+
+## Run Simulation Examples
+- ``` bash
+  cd CurricularAnalytics.jl/example/Simulations
+  ```
+- ``` bash
+  julia runSimulation.jl
+  ```
+
+## Set Course Pass Rate for a Specific Course
+- To access all courses in the curriculum:
+  - If the degree plan is stored in the variable `degreePlan`
+  - ``` julia
+    courses = degreePlan.curriculum.courses
+    ```
+  - First Course:
+    ``` julia
+    courses[1]
+    ```
+  - Change the pass rate of the first course to 0.5:
+    ``` julia
+    courses[1].passrate = 0.5
+    ```
+
+**Note:** Change the simulation configurations and degree plans by adjusting **config** section
```

Comment:
Could we set up a docs/src/simulation.md file and include some of this information there?

Reply:
It's done. 179c41190af4e3708a1c12d1494416deb9d7eb74